api/HTMLElement.json

```diff
@@ -3115,10 +3115,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLElement/spellcheck",
           "support": {
             "chrome": {
-              "version_added": "43"
```

Comment:
This came from https://github.com/mdn/browser-compat-data/pull/3502, also seemingly sourced from Confluence:
http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Chrome_42.0.2311.135_OSX_10.11.6%22,%22Chrome_42.0.2311.135_Windows_10.0%22,%22Chrome_43.0.2357.65_OSX_10.11.6%22,%22Chrome_43.0.2357.81_Windows_10.0%22%5D&q=%22spellcheck%22

Here it seems like the `spellcheck` attribute has moved from a number of specific interfaces up to `HTMLElement`, which means that before Chrome 43, `document.createElement('div').spellcheck` would have returned something different.

It's not a particularly interesting difference, but nonetheless I think we'll need to treat it as `partial_implementation` before Chrome 43.

Reply:
I just did a quick test, and it seems that `spellcheck` is actually present on _every_ HTML element, even unknown/made-up elements, so it's unclear to me why Confluence would report otherwise.

I wrote the following to test support:

```javascript
var tags = ['a', 'area', 'audio', 'base', 'basefont', 'body', 'br', 'button', 'canvas', 'content', 'data', 'datalist', 'details', 'dialog', 'div', 'dl', 'embed', 'fieldset', 'font', 'form', 'frame', 'frameset', 'head', 'h1', 'hr', 'html', 'iframe', 'img', 'input', 'isindex', 'keygen', 'label', 'legend', 'li', 'link', 'map', 'marquee', 'menu', 'menuitem', 'meta', 'meter', 'del', 'object', 'ol', 'optgroup', 'option', 'output', 'p', 'param', 'picture', 'pre', 'progress', 'blockquote', 'script', 'select', 'shadow', 'slot', 'source', 'span', 'style', 'caption', 'td', 'col', 'table', 'tr', 'tbody', 'template', 'textarea', 'time', 'title', 'track', 'ul', 'unknown', 'video', 'foo'];

var result = "";

for (var i = 0; i < tags.length; i++) {
	var tag = tags[i];

	var el = document.createElement(tag);
	var sc = el.spellcheck;

	if (!sc) {
		sc = "<span style='color: red;'>FALSE</span>";
	}

	result += tag + ": " + sc + "<br />";
}

var r = document.createElement('p');
r.innerHTML = result;
document.body.appendChild(r);
```

Reply:
I cherry-picked this as #7239 in efforts to reduce the size of this PR.

Reply:
Thanks!